### ISSUE TITLE
Is there a new version of this package?

### ISSUE THREAD

**Issue by JacksonGL** [jrm2k6/contwidgetor#4]
It seems that the authentication of this package uses SHA-1 ([code location](
https://github.com/jrm2k6/contwidgetor/blob/13b302d1d4b82b9ea7a4e226afa86a74f4d92ea3/src/data/bitbucket-commits-fetcher.js#L14)), which has been compromised ([link](https://security.googleblog.com/2017/02/announcing-first-sha1-collision.html?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+GoogleOnlineSecurityBlog+%28Google+Online+Security+Blog%29)) a few weeks ago.

Using SHA-256 would be safer.

**Comment by jrm2k6** [comment]
Do you have any issue using it?

**Comment by JacksonGL** [comment]
No. I was trying to report the potential security issue mentioned above. Not sure if this is the right place since it hasn't been updated since a few months ago.

**Comment by jrm2k6** [comment]
Oh yeah I saw that, I will try to fix it really soon

**Comment by JacksonGL** [comment]
Cool! Thanks.

**Comment by jrm2k6** [comment]
Sorry getting to this only now. 
For bitbucket I am using oauth 1.0, I am not sure SHA-256 is supported so I will have to investigate, or hopefully they have updated their api to let us use oauth2.0, meaning we could use SHA-256

**Comment by JacksonGL** [comment]
I see. Thanks for letting me know.
